projects/components/src/table/cells/data-renderers/time-ago/time-ago-table-cell-renderer.component.ts

```diff
@@ -10,7 +20,7 @@ import { TableCellAlignmentType } from '../../types/table-cell-alignment-type';
   styleUrls: ['./time-ago-table-cell-renderer.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="cell-value" [htTooltip]="this.value | htDisplayTimeAgo">
+    <div class="cell-value" [htTooltip]="this.value | htDisplayDate: this.dateFormat">
```

Comment:
Changed the tooltip back to timestamp. I think showing the underlying data is more important.